Just getting a bunch of unresolved modules

```ERROR in ./x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/Distribution/index.tsx
    Module not found: Error: Can't resolve '../../../shared/LoadingPrompt' in '/Users/casperhubertz/dev/elastic/kibana/x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/Distribution'

    ERROR in ./x-pack/legacy/plugins/apm/public/components/app/TransactionOverview/List/index.tsx
    Module not found: Error: Can't resolve '../../../shared/LoadingPrompt' in '/Users/casperhubertz/dev/elastic/kibana/x-pack/legacy/plugins/apm/public/components/app/TransactionOverview/List'

    ERROR in ./x-pack/legacy/plugins/apm/public/components/app/ServiceOverview/NoServicesMessage.tsx
    Module not found: Error: Can't resolve '../../shared/LoadingPrompt' in '/Users/casperhubertz/dev/elastic/kibana/x-pack/legacy/plugins/apm/public/components/app/ServiceOverview'

    ERROR in ./x-pack/legacy/plugins/apm/public/components/app/Settings/SettingsList.tsx
    Module not found: Error: Can't resolve '../../shared/LoadingPrompt' in '/Users/casperhubertz/dev/elastic/kibana/x-pack/legacy/plugins/apm/public/components/app/Settings'

    ERROR in ./x-pack/legacy/plugins/apm/public/components/app/TraceOverview/TraceList.tsx
    Module not found: Error: Can't resolve '../../shared/LoadingPrompt' in '/Users/casperhubertz/dev/elastic/kibana/x-pack/legacy/plugins/apm/public/components/app/TraceOverview'```

--- a/x-pack/legacy/plugins/apm/public/components/app/ServiceOverview/__test__/NoServicesMessage.test.tsx
+++ b/x-pack/legacy/plugins/apm/public/components/app/ServiceOverview/__test__/NoServicesMessage.test.tsx
@@ -10,12 +10,16 @@ import { NoServicesMessage } from '../NoServicesMessage';
 
 describe('NoServicesMessage', () => {
   it('should show only a "not found" message when historical data is found', () => {
-    const wrapper = shallow(<NoServicesMessage historicalDataFound={true} />);
+    const wrapper = shallow(
+      <NoServicesMessage isLoading={false} historicalDataFound={true} />
+    );
     expect(wrapper).toMatchSnapshot();
   });
 
   it('should show a "no services installed" message, a link to the set up instructions page, a message about upgrading APM server, and a link to the upgrade assistant when NO historical data is found', () => {
-    const wrapper = shallow(<NoServicesMessage historicalDataFound={false} />);
+    const wrapper = shallow(
+      <NoServicesMessage isLoading={false} historicalDataFound={false} />
+    );
     expect(wrapper).toMatchSnapshot();
   });
 });

--- a/x-pack/legacy/plugins/apm/public/components/app/Settings/SettingsList.tsx
+++ b/x-pack/legacy/plugins/apm/public/components/app/Settings/SettingsList.tsx
@@ -292,16 +292,18 @@ export function SettingsList() {
 
         <EuiSpacer size="m" />
 
-        <ManagedTable
-          noItemsMessage={
-            status === 'loading' ? <LoadingStatePrompt /> : emptyStatePrompt
-          }
-          columns={COLUMNS}
-          items={data}
-          initialSortField="service.name"
-          initialSortDirection="asc"
-          initialPageSize={50}
-        />
+        {status === 'success' && !hasConfigurations ? (
+          emptyStatePrompt
+        ) : (
+          <ManagedTable
+            noItemsMessage={<LoadingStatePrompt />}
+            columns={COLUMNS}
+            items={data}
+            initialSortField="service.name"
+            initialSortDirection="asc"
+            initialPageSize={50}
+          />
+        )}
       </EuiPanel>
     </>
   );

--- a/x-pack/legacy/plugins/apm/public/components/app/TraceOverview/TraceList.tsx
+++ b/x-pack/legacy/plugins/apm/public/components/app/TraceOverview/TraceList.tsx
@@ -4,7 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { EuiToolTip, EuiEmptyPrompt } from '@elastic/eui';
+import { EuiToolTip } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import React from 'react';
 import styled from 'styled-components';

--- a/x-pack/legacy/plugins/apm/server/routes/__test__/routeFailures.test.ts
+++ b/x-pack/legacy/plugins/apm/server/routes/__test__/routeFailures.test.ts
@@ -13,7 +13,7 @@ import { initTracesApi } from '../traces';
 describe('route handlers should fail with a Boom error', () => {
   let consoleErrorSpy: any;
 
-  async function testRouteFailures(init: (core: InternalCoreSetup) => void) {
+  function testRouteFailures(init: (core: InternalCoreSetup) => void) {
     const mockServer = { route: jest.fn() };
     const mockCore = ({
       http: {
@@ -44,7 +44,7 @@ describe('route handlers should fail with a Boom error', () => {
     };
 
     const routes = flatten(mockServer.route.mock.calls);
-    routes.forEach(async (route, i) => {
+    routes.forEach((route, i) => {
       test(`${route.method} ${route.path}"`, async () => {
         await expect(route.handler(mockReq)).rejects.toMatchObject({
           message: 'request failed',
@@ -65,15 +65,15 @@ describe('route handlers should fail with a Boom error', () => {
     consoleErrorSpy.mockRestore();
   });
 
-  describe('error routes', async () => {
-    await testRouteFailures(initErrorsApi);
+  describe('error routes', () => {
+    testRouteFailures(initErrorsApi);
   });
 
-  describe('service routes', async () => {
-    await testRouteFailures(initServicesApi);
+  describe('service routes', () => {
+    testRouteFailures(initServicesApi);
   });
 
-  describe('trace routes', async () => {
-    await testRouteFailures(initTracesApi);
+  describe('trace routes', () => {
+    testRouteFailures(initTracesApi);
   });
 });

